opm/autodiff/StandardWell_impl.hpp

```diff
@@ -583,6 +591,18 @@ namespace Opm
                     cq_s_poly *= extendEval(intQuants.polymerConcentration() * intQuants.polymerViscosityCorrection());
                 }
                 connectionRates_[perf][contiPolymerEqIdx] = Base::restrictEval(cq_s_poly);
+
+                if (this->has_polymermw) {
+                    if (well_type_ == PRODUCER) {
```

Comment:
I think a value should be assigned to `connectionRates_[perf][contiPolymerMWEqIdx]` also for injectors. If not, why?

Reply:
Injectors will be handled in `StandardWellV_impl`.